File: SIR1_java/src/enums/Level.java
package enums;

public enum Level {
    EASY,
    MEDIUM,
    HARD;

    public String getDescription(Level level) {
        switch (level) {
            case EASY:
                return "This recipe is very easy, you should try";
            case HARD:
                return "Very hard, be careful";
            case MEDIUM:
                return "This is more complicated, pay attention!";
            default:
                return "";
        }
    }
}

File: SIR1_kotlin/src/main/kotlin/enums/Level.kt
package enums

enum class Level {
    EASY,
    MEDIUM,
    HARD;

    fun getDescription(): String? {
        return when (this) {
            EASY -> "This recipe is very easy, you should try"
            MEDIUM -> "This is more complicated, pay attention!"
            HARD -> "Very hard, be careful"
        }
    }
}

